tensornetwork/backends/jax/jax_backend.py

```diff
@@ -49,4 +48,28 @@ def convert_to_tensor(self, tensor: Tensor) -> Tensor:
   def concat(self, values: Tensor, axis: int) -> Tensor:
     return numpy.concatenate(values, axis)
 
-  #TODO: add initializers (randn, zeros, eye, ones)
+  def randn(self,
+            shape: Tuple[int, ...],
+            dtype: Optional[numpy.dtype] = None,
+            seed: Optional[int] = None) -> Tensor:
+    if not seed:
+      seed = numpy.random.randint(0, 2**63)
+    key = self.jax.random.PRNGKey(seed)
+    if not dtype:
+      dtype = self.dtype
+    if not dtype:
```

Comment:
Why are you doing this check twice?

Reply:
self.dtype can also be None

Reply:
do `dtype = self.dtype if self.dtype is not None else numpy.float64` instead.